.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Daily build
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every day at 1:00 AM
+  workflow_dispatch: {}
+jobs:
+  release_image:
+    name: Build and publish docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_REPOSITORY: mongodb/atlas
+      PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
```

Reply:
How is this different? It seems to be the same

Reply:
> Set fetch-depth: 0 to fetch all history for all branches and tags

We only do that when we need whole repo history for very narrow use cases. It adds a lot of time to the job execution.

Reply:
This is needed to fetch the tags (tested it on my fork and it breaks without). In the latest run this step took 9 seconds so I think nothing to worry about in terms of execution time

Reply:
See my other comments. 
You are looking for valid releases with binaries attached. Looking for tags might catch us in the middle of the release etc.

Reply:
Since new approach is used we no longer need it?